Algorithmia/datafile.py

```diff
@@ -70,6 +71,18 @@ def getJson(self):
         # Make HTTP get request
         return self.client.getHelper(self.url).json()
 
+    def getNumpy(self):
+        exists, error = self.existsWithError()
+        if not exists:
+            raise DataApiError('unable to get file {} - {}'.format(self.path, error))
+        np_loader = pkgutil.find_loader('numpy')
```

Comment:
I didn't know about this `pkgutil`, nice trick to use here. That's why I love code reviews, thanks for the learning opportunity!

Reply:
No problem :) I just discovered this as well, definitely a better way than try/except